### ISSUE TITLE
Home page featured video should actually embed the video on the page

### ISSUE THREAD

**Issue by rbbeeston** [boostorg/website-v2#1153]
currently the home page featured video looks like this:
![Screenshot 2024-07-18 at 8 30 23 AM](https://github.com/user-attachments/assets/a339228b-1a81-43a0-a907-87f3901c1dca)

should look more like this:
![Screenshot 2024-07-18 at 8 38 22 AM](https://github.com/user-attachments/assets/669fa2f0-393e-41b3-a731-46e40819205a)

**Comment by sdarwin** [comment]
@rbbeeston previously we had been embedding videos.   It led to some controversy in the boost community since videos use cookies and/or require a GDPR popup asking about permission.   That is also why we removed Google Analytics and switched to Plausible analytics.

**Comment by vinniefalco** [comment]
Logged-in users should have the option to show embedded videos. Alternatively, we could give unregistered users a way to opt-out, by storing a permanent cookie on their computer (is that possible?)

**Comment by sdarwin** [comment]
The idea at the time was to avoid those GDPR popups and cookie permissions by not using third-party cookies.

Now, you are saying that you want to return to that?   How about Google Analytics?  Either way, then the cookie permissions stuff must be implemented.

**Comment by vinniefalco** [comment]
I am just complaining, without offering solutions or suggesting change.

**Comment by rbbeeston** [comment]
Was discussed during the weekly meeting, due to the amount of other coding to deal with GDPR issues, it's added to the backlog for now. Another alternative would be to add a thumbnail in lieu of the embedded video, but it's on hold until after more pressing issues are completed.

**Comment by rbbeeston** [comment]
I did a bit of research on this, I don't know if it's enough, but youtube has a "Privacy Enhanced Mode" for embedding links. However, there would be data stored if the video is actually played.

![Image](https://github.com/user-attachments/assets/a36dee2e-c522-4102-a810-59803c5b9f04)

**Comment by vinniefalco** [comment]
try a thumbnail generation service

**Comment by rbbeeston** [comment]
looks like we should be able to pull it using `https://i.ytimg.com/vi/<videoID>/hqdefault.jpg`

**Comment by vinniefalco** [comment]
Right, and to be clear we need our backend to make the request for the thumbnail and cache it ourselves. Otherwise, ytimg.com will be tracking user IPs.
